api/test/h3-data/h3-risk-map.spec.ts

```diff
@@ -34,8 +55,10 @@ describe('H3 Data Module (e2e) - Risk map', () => {
   let unitConversionRepository: UnitConversionRepository;
   const fakeTable = 'faketable';
```

Comment:
question for you and @alexeh : what value do these add? I don't really see a reason to have these constants, and I think code readability would improve if we just remove them

Reply:
Good point @tiagojsag . Since this aims to replace a magic table, the name does not really matter (IMO) and its at the main describe block scope to be reused in more than one tests, instead of adding a new string a instantiate a new constant in each test block. 

I wouldn't mind if you consider removing it and then use a new string when creating a fake h3 table is required